detector/src/main/java/com/synopsys/integration/detector/result/DetectorResult.java

```diff
@@ -42,4 +44,27 @@ public boolean getPassed() {
     public String getDescription() {
         return description;
     }
+
+    public String getStatusCode() {
+        if (!passed) {
+            return formatResultNameToStatusCode(resultName);
+        }
+        return "PASSED";
+    }
+
+    private String formatResultNameToStatusCode(String resultName) {
+        // Regular Expression
+        String regex = "([a-z])([A-Z]+)";
```

Comment:
Don't see why the regex needs to be on its own line and defined before it's used.

Reply:
agreed